lib/guardian/permissions/text_encoding.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Guardian.Permissions.TextEncoding do
+  @behaviour Guardian.Permissions.PermissionEncoding
+  use Bitwise
+
+  def encode(value, type, perm_set) when is_integer(value) do
+    perms = Map.get(perm_set, type)
+
+    for {k, v} <- perms, band(value, v) == v, into: [] do
+      k |> to_string()
```

Comment:
Please don't use pipelines for a single function call, this can be `to_string(k)` 👍